assets/js/validateForm.js

```diff
@@ -0,0 +1,43 @@
+function validateForm(validationSetUp, alert) {
+    let isValid = true; 
+    Object.keys(validationSetUp).forEach(field => {
+        const inputElement = document.getElementById(field);
+        const value = inputElement.value.trim();
+        const { validator, errorMessageI18n, idErrorElement } = validationSetUp[field];
+
+        if (!validator(value)) {
+            inputElement.classList.add('border-red-500');
+            inputElement.classList.remove('focus:border-primary-color');
+            const errorElement = document.getElementById(idErrorElement);
+            errorElement.textContent = t(errorMessageI18n);
+            errorElement.classList.remove('hidden');
+            isValid = false;
+        }
+        else {
```

Comment:
```suggestion
        } else {
```
Viết if else mình ưu tiên viết theo hướng thuận trước nha đặt điều kiện `validator(value)` nên trước thay vì `!validator(value)`